ui/react-components/components/DatePicker/DatePicker.jsx

```diff
@@ -35,10 +35,17 @@ const AppointmentDatePicker = (props) => {
            <DateInput value={dateInputValue}
                       minDate={minDate && minDate.toDate()}
                       isDisabled={isDisabled}
-                      onBlur={handleBlurOfDateInput}/>
+                      onBlur={handleBlurOfDateInput}
+                      hideDatePicker = {hideDatePicker}
+           />
            <div disabled={isDisabled} data-testid="datePicker-Calendar"> <DatePicker
                 selected={isDisabled? '' : selectedDate}
-                onChange={(date) =>onChange(datetoMomentDate(date))}
+                onSelect={(date) => {
```

Comment:
can we bring it out as a method and not have it inline?

Reply:
why  date.getTime() === selectedDate.getTime() is required?

Reply:
This is for, we are closing the date picker onChange of selected date. If i select the same date, onChange will not be called, in that case, we are checking if the same day is selected, it will close the date picker. I will move it to a separate function, not inline

Reply:
date.getTime() === selectedDate.getTime() will get time not date right?